test/v2/Jamfile.jam

```diff
@@ -27,6 +27,10 @@ project : requirements
   <target-os>freebsd:<linkflags>-lutil
   <target-os>freebsd:<linkflags>-lprocstat
   <target-os>freebsd:<cxxflags>-std=c++11
+  <target-os>bsd:<linkflags>-lpthread
+  <target-os>bsd:<linkflags>-lc
+  <target-os>bsd:<linkflags>-lkvm
+  <target-os>bsd:<cxxflags>-std=c++11
```

Comment:
Nope, wrong place. I think -lkvm should suffice, actually.

Reply:
Of course also wrong above for freebsd.

Reply:
Run `b2 cxxstd=11` from the command line.

Reply:
That might be true with -pthread, but I know for certain from experience on *BSD's they outright do not link to -lc without the developer doing so manually.